content/release-10-15-0/platform-10-15-0-bundle/10-15-0-152.md

```diff
@@ -46,6 +46,14 @@ Issue</th>
 </tr>
 </thead><tbody>
 
+<tr>
+<td>
+Inventory</td>
+<td>The cached content-length header value is now always added only for HTTP/1.0 for the binary download endpoint <code>GET /inventory/binaries/{moID}</code>. This is done to correctly support the chunked Transport-Encoding for HTTP/1.1+ and fixes the issue where devices could not upgrade firmware because of the missing Transport-Encoding=chunked header.</td>

```

Comment:
is it "content-length" or "content length" ?

Reply:
afaik it is a proper name